### DISCUSSION TITLE
funding vs sponsor as contributing organization

### DISCUSSION THREAD

**Discussion by h-mayorquin** [dandi/helpdesk#38] [Q&A]
Hi, so we are trying to add the following organizations as contributors in a dataset:
* Nakajima Foundation.
* NIH (G.B.)
* National Science Foundation (to Euisik Yoon)

And while I am adding this the interface of dandi offers me two options for dcite that seem equally plausible to me: dcite:Funder and dcite:Sponsor.

My question is? Are they redundant or are they mean to cover different semantics?

Kind regards,

**Comment by satra** [comment]
@h-mayorquin - this is mostly personal preference - i tend to use funder for organizations providing monetary awards (NIH, NSF, foundations), etc.,. while i use sponsor for in-kind or other types of sponsorship.

**Reply by h-mayorquin** [reply]
Thanks, along those lines, how do add specific grants? I have seen that some projects add an organization and then an identifier for an award to declare the grant number / identifier. Is this the intended use?

**Reply by satra** [reply]
indeed, when you add an organization as a contributor there is a field for award identifier, and the role clarifies the organizations contribution.
